src/Collection/CollectionInterface.php

```diff
@@ -915,10 +915,27 @@ public function zipWith($items, $callable);
      * ```
      *
      * @param int $chunkSize The maximum size for each chunk
-     * @param boolean $preserveKeys If the keys of the array should be preserved
      * @return \Cake\Collection\CollectionInterface
+     * @deprecated 4.0.0 Deprecated in favor of chunks
      */
-    public function chunk($chunkSize, $preserveKeys = false);
+    public function chunk($chunkSize);
+
+    /**
+     * Breaks the collection into smaller arrays of the given size.
+     *
+     * ### Example:
+     *
+     * ```
+     * $items [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11];
+     * $chunked = (new Collection($items))->chunks(3)->toList();
+     * // Returns [[1, 2, 3], [4, 5, 6], [7, 8, 9], [10, 11]]
+     * ```
+     *
+     * @param int $chunkSize The maximum size for each chunk
+     * @param bool $preserveKeys If the keys of the array should be preserved
+     * @return \Cake\Collection\CollectionInterface
+     */
+    public function chunks($chunkSize, $preserveKeys = false);
```

Comment:
Most of the method names of this interface are verbs: take, skip, filter, append etc. So `chunks()` would be oddly named.